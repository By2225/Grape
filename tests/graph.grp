fun Int main() {
	Graph<Int, Int> a;

        a = <<'3' -3- '4'>>;

	return 0;
}
