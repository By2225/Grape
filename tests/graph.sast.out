
fun Int main() {
Graph<Node<Int>, Edge<Int>> a;
(Graph<Node<Int>, Edge<Int>> : a = (Graph<Node<Int>, Edge<Int>> : <(Node<Int> : '(Int : 3)') (Edge<Int> : _(Int : 3)_<) (Node<Int> : '(Int : 4)') (Void : )>));
return (Int : 0);
}
